public_html/composer.json

```diff
@@ -0,0 +1,129 @@
+{
+	"name": "wordcamp/wordcamp.org",
+	"description": "",
+	"homepage": "https://wordcamp.org",
+	"license": "GPL-2.0-or-later",
+	"support": {
+		"issues": "https://github.com/WordPress/wordcamp.org/issues"
+	},
+	"config": {
+		"platform": {
+			"php": "7.2"
+		},
+		"vendor-dir": "wp-content/mu-plugins/vendor"
+	},
+	"repositories": [
+		{
+			"type": "composer",
+			"url": "https://wpackagist.org/"
```

Comment:
`MIGHT` - I'm a bit leery of this, mainly for [security reasons](https://github.com/outlandishideas/wpackagist/issues/169).

It may not be a big deal, though, since prod will never use it. Locally, committers will have the SVN externals locally,  so we'd spot any differences. Although, if we use the `latest.zip` here, like suggested below, then that might be an issue. I guess we'd just `svn revert` after `composer update`, though, so maybe not.

What's the advantage of using WPackagist over just specifying the URL directly? Is it that we don't have to also specify that they should be installed to `wp-content/plugins` ?

Reply:
Since this is only for local dev, and not for production, I think checking the lock file is sufficient, as [this comment explains](https://github.com/outlandishideas/wpackagist/issues/169#issuecomment-307415860).

The advantage is that Composer treats wpackagist as an actual Composer package repository, so the packages that come from there only need to be referenced once in the `require` or `require-dev` property, instead of also having to define a repository and therefore update the version number in three different places for a given package each time it upgrades.

Sidenote, checksums have been implemented for plugins and themes now, haven't they? It doesn't look like Outlandish has added anything related to that, despite [this comment](https://github.com/outlandishideas/wpackagist/issues/169#issuecomment-350897739).

Reply:
👍 , that WFM, as long as we document that it should never be used on prod/remote sandboxes.

Reply:
Private devdocs updated to reflect this.